src/Glue.elm

```diff
@@ -214,31 +220,8 @@ updateModel (Glue rec) fc msg model =
     rec.set (fc msg <| rec.get model) model
 
 
-{-| Trigger Cmd in child's function
-
-_Commands are async. Therefore trigger doesn't make any update directly.
-Use [`updateModel`](#updateModel) over `trigger` when you can._
-
-    -- Child module
-    triggerEmit : Counter.Model -> Cmd Counter.Msg
-    triggerEmit model ->
-        Task.perform identity <| Task.succeed <| Counter.Emit model
-
-    -- Parent module
-    update : Msg -> Model -> ( Model, Cmd Msg )
-    update msg model =
-        case msg of
-            IncrementCounter ->
-                ( model, Cmd.none )
-                    |> Glue.trigger counter triggerIncrement
-
--}
-trigger : Glue model subModel msg subMsg -> (subModel -> Cmd subMsg) -> ( model, Cmd msg ) -> ( model, Cmd msg )
-trigger (Glue rec) fc ( model, cmd ) =
-    ( model, Cmd.batch [ Cmd.map rec.msg <| fc <| rec.get model, cmd ] )
```

Comment:
This is used in examples. I'm going to add it back for now. We already did a lot of work on this,